### DISCUSSION TITLE
How do i use Implicit relations for update in Prisma upsert

### DISCUSSION THREAD

**Discussion by Prajwal056** [prisma/prisma#16793] [Q&A]
The following code should create a new id if id is not found, if id is found it should update. for that i have used upsert function function.

in the model (prisma) i am using https://www.prisma.io/docs/guides/database/troubleshooting-orm/help-articles/working-with-many-to-many-relations my model has an Implicit relations. how do i use update in Implicit relations
```
const data = await PRISMA.ref_types.upsert({
                    where: { id: id },
                    update: {
                        business_name: business_name,
                        business_logo: business_logo,
                        business_type: business_type,
                        timezone: timezone,
                        location_contact_details: { update: { contact: contact, email: email, } },
                        location_addresses: { update: { address: address, location_id: req.body.location_id, } },
                        languages: { create: { languages: languages } },
                        ref: headerData?.ref,
                    },
                    create: {
                        business_name: business_name,
                        business_logo: business_logo,
                        business_type: business_type,
                        timezone: timezone,
                        location_contact_details: { create: { contact: contact, email: email, } },
                        location_addresses: { create: { address: address, location_id: req.body.location_id, } },
                        languages: { create: { languages: languages } },
                        ref: headerData?.ref,
                    },
                })`

**Comment by nurul3101** [comment]
Hey @Prajwal056 👋 
Can you please share your schema file so that I can understand your use case better?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
